docs/tempo/website/getting-started/instrumentation.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Instrumentation
+title: Instrumentation references
```

Comment:
Should we also change the ref to this page in `_index.md` ?  Currently it is `[instrumentation examples]({{< relref "./instrumentation" >}})`

Reply:
Done. Good catch.